__tests__/plugin.test.ts

```diff
@@ -1582,4 +1582,18 @@ describe('babel plugin', () => {
       expect(code).toMatchSnapshot();
     });
   });
+
+  it('skips initData on web', () => {
+    process.env.REANIMATED_BABEL_PLUGIN_IS_WEB = '1';
+    const input = html`<script>
+      function foo() {
+        'worklet';
+        var foo = 'bar';
+      }
+    </script>`;
+
+    const { code } = runPlugin(input);
+    expect(code).toHaveWorkletData(0);
+    expect(code).toMatchSnapshot();
+  });
```

Comment:
Make sure to clean-up after a test. This way it's left all future tests will not include initData. 
For reference see https://github.com/software-mansion/react-native-reanimated/blob/main/__tests__/plugin.test.ts#L1441

Also the name of the test is somehow misleading. In current implementation, our plugin doesn't know on what platform it will bundle - we have to specify this explicitly (and we can do this regardless of platform!).

Therefore the name of the test should be more like `doesn't include initData when REANIMATED_BABEL_PLUGIN_IS_WEB is set` (with current implementation)